pages/common/helm.md

```diff
@@ -0,0 +1,35 @@
+# helm
+
+> [Helm](https://helm.sh/) is a package manager for Kubernetes.
```

Comment:
tldr is a command line tool. So markdown syntax like this does not work. Just say Helm